common_features.mk

```diff
@@ -295,8 +295,26 @@ ifeq ($(strip $(LEADER_ENABLE)), yes)
   OPT_DEFS += -DLEADER_ENABLE
 endif
 
+ifeq ($(strip $(JOYSTICK_ENABLE)), yes)
+    OPT_DEFS += -DJOYSTICK_ENABLE
+    SRC += $(QUANTUM_DIR)/process_keycode/process_joystick.c
+    SRC += $(QUANTUM_DIR)/joystick.c
+    ifeq ($(PLATFORM),AVR)
+      SRC += drivers/avr/analog.c
+    endif
+endif
+
 include $(DRIVER_PATH)/qwiic/qwiic.mk
 
+ifeq ($(strip $(JOYSTICK_ENABLE)), yes)
```

Comment:
Looks like this got added twice?

Reply:
This block looks to be identical, and added twice by accident.

One of them should be removed.